plugins/com.google.cloud.tools.eclipse.dataflow.ui.test/src/com/google/cloud/tools/eclipse/dataflow/ui/launcher/DefaultedPipelineOptionsComponentTest.java

```diff
@@ -136,7 +135,7 @@ public void testDefaults_values() {
 
     assertTrue(component.isUseDefaultOptions());
     Map<String, String> values = component.getValues();
-    assertEquals("pref-email", values.get(DataflowPreferences.ACCOUNT_EMAIL_PROPERTY));
+    assertEquals(TestAccountProvider.EMAIL_ACCOUNT_1, values.get(DataflowPreferences.ACCOUNT_EMAIL_PROPERTY));
```

Comment:
Dataflow preferences will not be able to set an email anymore. Only the one from the gcloud CLI will be picked up